lib/statement.rb

```diff
@@ -0,0 +1,38 @@
+class Statement 
+
+  attr_reader :tracker
+
+  def initialize(tracker = Tracker.new)
+    @tracker = tracker
+  end
+
+  def compose_line(date, credit, debit, balance)
+    line = []
+    line[0] = date
+    line[1] = credit.nil? ? "" : credit
+    line[2] = debit.nil? ? "" : debit
+    line[3] = balance
+    line.join(" || ")
+  end
+
+  def compose_body
+    body = []
```

Comment:
Is it possible to use `map` here to avoid having to explicitly make the body array?